src/oss/langchain/multi-agent.mdx

```diff
@@ -298,13 +299,21 @@ const callSubagent1 = tool(
 
 ## Handoffs
 
-In **handoffs**, agents can directly pass control to each other. The “active” agent changes, and the user interacts with whichever agent currently has control.
+In **handoffs**, agents pass control to each other. There are two main approaches:
 
-Flow:
+| Approach | How it works | Best for |
+|----------|--------------|----------|
+| [**Tool handoffs**](#tool-handoffs) | Agent calls a tool that updates state to transfer control | Dynamic routing, user-driven flows |
+| [**Graph handoffs**](#graph-handoffs) | Graph edges define transitions between agents | Fixed pipelines, deterministic routing |
```

Comment:
langgraph is great for things that are dynamic and include cycles and bespoke logic, so "fixed pipelines, determinsitic routing" isn't really correct?